server/app/index.js

```diff
@@ -11,7 +11,7 @@ module.exports = function (db) {
 
     // Routes that will be accessed via AJAX should be prepended with
     // /api so they are isolated from our GET /* wildcard.
-    app.use('/api', require('./routes'));
+    app.use('/', require('./routes/'));
```

Comment:
I'm imagining "routes" as holding all of our routes, rather than only our api routes.